architecture/core_concepts/routes.adoc

```diff
@@ -1183,3 +1185,69 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+[[disable-namespace-ownership-check]]
+== Disabling the Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace `ns1` creates the oldest route r1 with host `*_www.abc.xyz_*`, then it
+owns that host name and subdomain (`abc.xyz`) for wildcard routes.  If another
+namespace `ns2` tries to create a route with say a different path `*_www.abc.xyz/path1/path2_*`,
+it would fail because a route in another namespace (`ns1` in this case) owns
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.adoc#using-wildcard-routes[wildcard routes]
+as the namespace that owns the subdomain owns all hosts in the subdomain.
+If a namespace owns subdomain `*.abc.xyz*`, anouther namespace can't claim `z.abc.xyz`.
```

Comment:
@Miciah I am going to leave this alone and address it in the general revisions in pr3943. This feels like the discussion needs to be consolidated in one place and greatly improved. There are bits of this spread around in the docs. This pr was intended to be a fairly quick fix for the problem at hand. You have very rightly pointed out that a lot more needs to be done. If I do it here the merge into pr 3943 will be grim.